stock_indicators/indicators/starc_bands.py

```diff
@@ -35,6 +36,10 @@ def get_starc_bands(quotes: Iterable[Quote], sma_periods: int = 20,
          - [STARC Bands Reference](https://python.stockindicators.dev/indicators/StarcBands/#content)
          - [Helper Methods](https://python.stockindicators.dev/utilities/#content)
     """
+    if sma_periods is None:
+        warn('The default value of sma_periods will be removed in the next version. Pass sma_periods explicitly.', DeprecationWarning, stacklevel=2)
```

Comment:
@DaveSkender 
found the way can notice the users. If the warning message is weird, please correct it.